projects/ngx-formentry/src/components/machine-learning-button/model-helper.ts

```diff
@@ -330,7 +330,7 @@ export function generatePredictionPayload(
 
   // convert months since last test
 
-  if(testHistory == 1065) { // Tested for HIV before?
+  if(testHistory == concepts.YES) { // Tested for HIV before?
```

Comment:
Does this use uuid or concept ID or either? We can adopt a standard approach to use the UUID if that works.

Reply:
Ok